Dockerfile

```diff
@@ -30,17 +30,15 @@ ADD requirements.txt requirements.txt
 RUN pip install -r requirements.txt && \
     rm -rf ~/.cache/pip
 
-# Installing FMRIPREP
+# Installing fitlins
 COPY . /root/src/fitlins
 RUN cd /root/src/fitlins && \
     pip install .[all] && \
     rm -rf ~/.cache/pip
 
 RUN ldconfig
 
-WORKDIR /root/src/fmriprep
-
-ENTRYPOINT ["/opt/conda/bin/fitlins"]
```

Comment:
Why drop the entrypoint?

Reply:
No good reason aside from the fact that it wasn't working, because /opt/conda/bin does not exist. 

In other words, doesn't seem like fitlins is being installed as an executable.

Reply:
Really? It works when I build locally. I guess I'll add this to the test.

Reply:
Yeah that was it. I added it back it and it works